bin/benchpark

```diff
@@ -153,15 +187,14 @@ def benchpark_setup_handler(args):
     instructions = f"""\
 To build/run these benchmarks, do the following:
 
-    cd {workspace_dir}
+    cd {workspace_dir}/workspace
 
     . {workspace_dir}/spack/share/spack/setup-env.sh
     . {workspace_dir}/ramble/share/ramble/setup-env.sh
 
     export SPACK_DISABLE_LOCAL_CONFIG=1
 
     ramble -D . workspace setup
-    ramble -D . on
```

Comment:
Why remove this? This runs the experiments.

Reply:
I removed it because it is in a separate step in the docs (easier to explain as a separate logical step), and also if people were to paste the whole thing, they'd be invoking the run even in the previous line exited with an error...

Reply:
Fair enough, although it does give a complete picture to include it.

If this stays deleted, the above line should be amended:

> To build/run these benchmarks, do the following:

(remove "run", possibly indicate that further steps are needed to run them)